src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2705,7 +2705,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>This warning occurs if the assembly attributes AssemblyKeyFileAttribute or AssemblyKeyNameAttribute found in source conflict with the /keyfile or /keycontainer command line option or key file name or key container specified in the Project Properties.</value>
   </data>
```

Comment:
I suggest to do a string search/replace over the modified error strings in the tests project, to make sure that the error comments in tests are updated as well.